tssc/step_implementers/tag_source/git.py

```diff
@@ -117,27 +117,42 @@ def _git_url(runtime_step_config):
         return return_val
 
     @staticmethod
-    def _git_tag(git_tag_value): # pragma: no cover
+    def _git_tag(git_tag_value):  # pragma: no cover
         try:
             # NOTE:
             # this force is only needed locally in case of a re-reun of the same pipeline
             # without a fresh check out. You will notice there is no force on the push
             # making this an acceptable work around to the issue since on the off chance
             # actually orverwriting a tag with a different comment, the push will fail
             # because the tag will be attached to a different git hash.
-            sh.git.tag(git_tag_value, '-f')
+            print(
+                sh.git.tag(  # pylint: disable=no-member
+                    git_tag_value,
+                    '-f'
+                )
+            )
         except sh.ErrorReturnCode:  # pylint: disable=undefined-variable
             raise RuntimeError('Error invoking git tag ' + git_tag_value)
 
     @staticmethod
-    def _git_push(url=None): # pragma: no cover
+    def _git_push(url=None):  # pragma: no cover
         try:
             if url:
-                sh.git.push(url, '--tag')
+                print(  # pylint: disable=no-member
+                    sh.git.push(
+                        url,
+                        '--tag'
+                    )
```

Comment:
@ppremru do we not need `_out=sys.stdout` here as a param to `sh.git` like the other `sh.*` commands in the other step impelementers?